<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>954</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="chatFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QTextEdit" name="textEdit">
                <property name="cursor" stdset="0">
                 <cursorShape>IBeamCursor</cursorShape>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="messageHorizontalLayout">
            <item>
             <widget class="QLineEdit" name="messageToSendLineEdit">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="clearButtonEnabled" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sendPushButton">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="buddyListFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="buddyListLabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>13</height>
           </size>
          </property>
          <property name="text">
           <string>Buddy List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>3</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="buddyScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>293</width>
             <height>857</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAdd_Peers"/>
    <addaction name="actionAdd_Buddies"/>
    <addaction name="actionSet_Keys"/>
    <addaction name="separator"/>
    <addaction name="actionView_Tracking"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Peers">
   <property name="text">
    <string>Add Peers</string>
   </property>
  </action>
  <action name="actionAdd_Buddies">
   <property name="text">
    <string>Add Buddies</string>
   </property>
  </action>
  <action name="actionSet_Keys">
   <property name="text">
    <string>Set Keys</string>
   </property>
  </action>
  <action name="actionView_Tracking">
   <property name="text">
    <string>View Tracking</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
